rpc/legacyrpc/methods.go

```diff
@@ -442,7 +406,7 @@ func addMultiSigAddress(icmd interface{}, w *wallet.Wallet, chainClient *dcrrpcc
 		return nil, err
 	}
 
-	err = chainClient.LoadTxFilter(false, []dcrutil.Address{p2shAddr}, nil)
+	err = s.chainClient.LoadTxFilter(false, []dcrutil.Address{p2shAddr}, nil)
```

Comment:
this is now a data race.  the chainClient field can be changed at any time.  The old version did not have this race since a copy of the pointer to the client was passed in as an argument.

Even if this is protected by a mutex, we could still hit logic races that cause nil pointer dereferences, so a nil check must also be performed here if doing it this way.